src/main/java/graphql/execution/instrumentation/dataloader/ChainedDataLoader.java

```diff
@@ -0,0 +1,58 @@
+package graphql.execution.instrumentation.dataloader;
+
+import graphql.ExperimentalApi;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.function.Function;
+
+
+@ExperimentalApi
+public final class ChainedDataLoader<T, U> extends CompletableFuture<U> {
```

Comment:
i feel like this is the wrong package for this this class

Its nothing to do with `instrumentation`

I suggest a new one `graphql.dataloader`